manifests/init.pp

```diff
@@ -46,7 +46,10 @@
   $url,
   $destination_directory,
   $destination_file = '',
-  $proxyAddress=''
+  $proxyAddress='',
```

Comment:
So `$proxyAddress` was camel case all along, but I would prefer underscore downcase names, just for the reason that they are more in keeping with Puppet practices. Thoughts?